Jenkinsfile-Release

```diff
@@ -36,33 +36,31 @@ dockerizedBuildPipeline(
   deploy: {
     final branch = gitBranch(env)
 
-    sh """
-      git config user.email "sonatype-ci@sonatype.com"
-      git config user.name "Sonatype CI"
-
-      git checkout ${branch}
-      git add docs
-      git commit -m 'Release Update'
-    """
+    runSafely 'git config user.email "sonatype-ci@sonatype.com"'
+    runSafely 'git config user.name "Sonatype CI"'
+    runSafely "git checkout ${branch}"
+    runSafely 'git add docs'
+    runSafely 'git commit -m "Release Update"'    
 
     sshagent(credentials: [sonatypeZionCredentialsId()]) {
-      sh 'git push'
+      runSafely 'git push'
     }
   },
   postDeploy: {
-    def chart = params.chart
-    def indexFile = readYaml file: 'docs/index.yaml'
-    def version = indexFile.entries[chart][0].version
-
+    // Verify Index.yaml
+    String chart = params.chart    
+    String version = verifyIndexYaml(chart)
+    
+    // Set Jira Fix Version
     jira.versionPrefix = jiraVersionMappings[chart]
-    jiraSetFixVersion(jira, version)
-
-    sleep(time: 60) // seconds
+    jiraSetFixVersion(jira, version)    
 
-    verifyDownloadLinks(
```

Comment:
For helm charts, a better link check is to confirm the `index.yaml` file is present and also check it has the latest version, so I am removing the `verifyDownloadLink` call and I have created the new method `verifyIndexYaml`

Reply:
why not check both index.yaml _and_ the tgz?

Reply:
Actually, you are right, we could be sure about both, I am going to add the check for the tgz.

Reply:
Done